src/openrct2-ui/windows/ServerList.cpp

```diff
@@ -463,17 +463,30 @@ static void window_server_list_scrollpaint(rct_window* w, rct_drawpixelinfo* dpi
 
         screenCoords.x = 3;
 
-        // Draw server information
-        if (highlighted && !serverDetails.Description.empty())
+        // Before we draw the server info, we need to know how much room we'll need for player info.
+        char players[32];
+        players[0] = 0;
```

Comment:
I just moved this from its original position below, but we might as well refactor it into something like
```suggestion
        char players[32] = { 0 };
```

Reply:
What's holding you back? It's a very minor change and unlikely to hinder code review.

Reply:
Not being on my computer, for starters.